src/main.cpp

```diff
@@ -43,20 +44,76 @@ Napi::Value DDWAF::GetDisposed(const Napi::CallbackInfo& info) {
 
 DDWAF::DDWAF(const Napi::CallbackInfo& info) : Napi::ObjectWrap<DDWAF>(info) {
   Napi::Env env = info.Env();
-  if (info.Length() < 1) {
-    Napi::Error::New(env, "Wrong number of arguments, expected 1").ThrowAsJavaScriptException();
+  size_t arg_len = info.Length();
+  if (arg_len < 1) {
+    Napi::Error::New(env, "Wrong number of arguments, expected at least 1").ThrowAsJavaScriptException();
     return;
   }
   if (!info[0].IsObject()) {
     Napi::TypeError::New(env, "First argument must be an object").ThrowAsJavaScriptException();
     return;
   }
+
+  ddwaf_config waf_config;
+  waf_config.limits.max_container_size = DDWAF_MAX_CONTAINER_SIZE;
+  waf_config.limits.max_container_depth = DDWAF_MAX_CONTAINER_DEPTH;
```

Comment:
@Anilm3 I am not sure why this is needed?

Reply:
Without these values initialized, the waf returns `DDWAF_ERR_INVALID_OBJECT` with large objects. Is this expected ?

Reply:
Setting all limits to 0 will signal the WAF to use defaults:

`ddwaf_config waf_config{{0, 0, 0}, {nullptr, nullptr}};`